Spec: Update mediaType to type for catalogEntry and add known types (#37)
Agent-Card/ai-card: specification/ai-catalog.md

    {
      "identifier": "urn:example:skill:code-review",
      "displayName": "Code Review Assistant",

>       "type": "application/ai-skills+zip",

Reviewer: I see this _was_ `agentskill`, but was changed to `ai-skills`.  Was that intended?

Author: Changed back to agent-skills+zip now. Thanks @jonathanhefner

`displayName`
: A string containing a human-readable name for the artifact.

`type`
: A string containing the identifier that specifies the type of the
  referenced artifact. This field is an open text format, so any string value is accepted. However, to ensure interoperability, it is RECOMMENDED to use one of the following recognized "known types" in the ecosystem when applicable:

  - `application/a2a-agent-card+json` — an A2A Agent Card
  - `application/mcp-server+json` — an MCP Server Card
  - `application/ai-catalog+json` — a nested AI Catalog
  - `application/ai-skills+tgz` — an AI Skill bundle (compressed tarball)
  - `application/ai-skills+zip` — an AI Skill bundle (ZIP archive)
  - `application/ai-skills+md` — an AI Skill defined in Markdown

>   These values combine the artifact type with the payload format. For any new or custom types not listed here, it is up to the specific client implementation to handle them correctly.

Reviewer 2: Should we split the ai-catalog/ai-card managed media types, from the third party ones? I was envisioning something like:

```
AI Catalog types:
application/ai-catalog+json - nested AI Catalog
application/agent-card+json - reserved Agent Card media type

Third Party Types:
(A2A, MCP etc)
```

Author: Sounds good. Updated.

`displayName`
: A string containing a human-readable name for the artifact.

`type`
: A string containing the identifier that specifies the type of the
  referenced artifact. This field is an open text format, so any string value is accepted. However, to ensure interoperability, it is RECOMMENDED to use one of the following recognized "known types" in the ecosystem when applicable, partitioned by their respective governance boundaries:

  **Core Protocol Types (Governed by the AI Catalog WG):**
  - `application/ai-catalog+json` — a nested AI Catalog
  - `application/agent-card+json` — reserved for a generic Agent Card format

  **Integrated Ecosystem & Third-Party Types (Governed externally):**
  - `application/a2a-agent-card+json` — an A2A Agent Card
  - `application/mcp-server+json` — an MCP Server Card
  - `application/ai-skills+zip` — an AI Skill bundle (ZIP archive)
  - `application/ai-skills+gzip` — an AI Skill bundle (gzipped tarball)

>   - `text/markdown; profile=ai-skill` — an AI Skill defined in a standard Markdown file

Reviewer 3: Profiles are expected to be a URI https://datatracker.ietf.org/doc/html/rfc6906#section-3

Author: Thanks for the information. Yes you're right. In that case the type looks a little long here. Let me summarize different options below.

>   - `text/markdown; profile=ai-skill` — an AI Skill defined in a standard Markdown file

Reviewer: ```suggestion
  - `application/agent-skills+zip` — an Agent Skill bundle (ZIP archive)
  - `application/agent-skills+gzip` — an Agent Skill bundle (gzipped tarball)
  - `text/markdown; profile=agent-skill` — an Agent Skill defined in a standard Markdown file
```

>   These values are designed to align with official IANA media type registration standards. Standard ecosystem types use registered structured syntax suffixes (`+json`, `+zip`, `+gzip`). For generic payload formats (like `text/markdown`), the standard `profile` parameter is used to specify the semantic schema (e.g., `profile=ai-skill`). For any new or custom types not listed here, it is up to the specific client implementation to handle them correctly.

Reviewer: ```suggestion
  These values are designed to align with official IANA media type registration standards. Standard ecosystem types use registered structured syntax suffixes (`+json`, `+zip`, `+gzip`). For generic payload formats (like `text/markdown`), the standard `profile` parameter is used to specify the semantic schema (e.g., `profile=agent-skill`). For any new or custom types not listed here, it is up to the specific client implementation to handle them correctly.
```

      {
        "identifier": "urn:claude-plugin:anthropic:example-plugin:mcp",
        "displayName": "Example Plugin MCP Server",
        "type": "application/mcp-server+json",
        "url": "https://github.com/anthropics/claude-plugins-official/blob/main/plugins/example-plugin/server-card.json"
      },
      {
        "identifier": "urn:claude-plugin:anthropic:example-plugin:skills",
        "displayName": "Example Plugin Skills",

>         "type": "application/ai-skills+zip",

Reviewer: ```suggestion
        "type": "application/agent-skills+zip",
```
